mozjexl/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2015 TechnologyAdvice
```

Comment:
I think we want to update this to something like:

Copyright for portions of mozJexl are held by TechnologyAdvice, 2015 as part of Jexl. All other copyright for  mozJexl are held by the Mozilla Foundation, 2017.